components/PersonView/PersonDetails.jsx

```diff
@@ -80,9 +80,13 @@ const PersonDetails = ({
                 <div className="govuk-summary-list__row">
                   <dt className="govuk-summary-list__key">Phone Number</dt>
                   <dd className="govuk-summary-list__value">
-                    {Object.values(phoneNumber[0]).map((number) => (
-                      <p key={number}>{number}</p>
-                    ))}
+                    <ul>
+                      {phoneNumber.map(({ phoneNumber, phoneType }) => (
+                        <li className="govuk-list" key={phoneNumber}>
```

Comment:
`govuk-list` should be on the `ul`, see reference at https://design-system.service.gov.uk/styles/typography/

Reply:
done now. doesn't seem to make any difference.

Reply:
yes, it's more because it's cleaner to set it to the parent rather than in all the children :)